src/styles/theme/color.ts

```diff
@@ -43,6 +43,7 @@ export const oakColorTokens = {
   magenta: "#d02aa7",
   purple: "#845ad9",
   teal: "#037b7d",
+  blackSemiTransparent: "#22222240",
```

Comment:
Added with agreement from UI team. Needed because if you use opacity on the backdrop it disables the backdrop blur filter.